src/components/IstioWizards/IstioWizard.tsx

```diff
@@ -109,8 +109,8 @@ class IstioWizard extends React.Component<Props, State> {
         const headerName = match.substring(i0 + 1, j0);
         const i1 = match.indexOf(' ', j0 + 1);
```

Comment:
Why not add .trim() to headerName too?

Reply:
Ah, probably I thought it was not necessary, let me check and update it.